source/globalCommands.py

```diff
@@ -4775,10 +4775,9 @@ def script_decreaseApplicationsVolume(self, gesture: "inputCore.InputGesture") -
 	@script(
 		description=_(
 			# Translators: Describes a command.
-			"Toggles other applications volume adjuster status",
+			"Allow or disallow to control the volume of other applications",
```

Comment:
In general, "allow or disallow to control", is not an EFL phrase. You would say "Allow or disallow controlling", or "Allow or disallow control of/over".
Or, as I think we more commonly use: "Enable/Disable control of/over".

Yet another way to say it, is what I have below.
```suggestion
			"Toggle whether control of other applications' volume is enabled.",
```